build/build-release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+readonly REPO_ROOT=$(git rev-parse --show-toplevel)
+
+sudo rkt run --uuid-file-save=rkt.uuid \
```

Comment:
As much as I like it, should we consider using a multi-stage Dockerfile instead of docker+rkt?